README.md

```diff
@@ -118,9 +118,23 @@ You can access snapshot builds from the sonatype snapshot repository by adding t
 ```
 
 ## Build
+
+### Requirements
+
+* [Node.Js](https://nodejs.org/en/download/)
+* Java JDK >= 1.8
+* [Lombok Java](https://projectlombok.org) Library (debian pacakge `liblombok-java`)
+* Maven, with the plugins:
+    * [JavaDoc](https://maven.apache.org/plugins/maven-javadoc-plugin/) (debian package `libmaven-javadoc-plugin-java`)
```

Comment:
The projects ships a maven wrapper. No need for an extra package.

Reply:
Right; sorry for the confusion. My build error occurred in a component using the library but it was my missing liblombok that was the issue.